src/components/MoneyReportHeader.tsx

```diff
@@ -208,8 +209,10 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
 
     const shouldDisableSubmitButton = shouldShowSubmitButton && !isAllowedToSubmitDraftExpenseReport(moneyRequestReport);
     const isFromPaidPolicy = policyType === CONST.POLICY.TYPE.TEAM || policyType === CONST.POLICY.TYPE.CORPORATE;
+
+    const hasDuplicate = hasDuplicateTransactions(moneyRequestReport?.reportID);
```

Comment:
Nitpicky comment, but I would rename this to `hasDuplicates` in plural since it makes more sense